ApollosProject/apollos-apps
# Chore tv client graphics

## ✏️ Description
- updates tv screens to show proper church logo



## 🔬 To Test
1. run `setup-tv.sh` script
2. Navigate around different screens
3. Should see proper logo for church that was set up

## 📸 Screenshots

![image](https://user-images.githubusercontent.com/7557597/216689507-30d6abf7-57be-48ad-97e0-05770e0cdf90.png)
![image](https://user-images.githubusercontent.com/7557597/216689765-90d48ae0-6295-4d7b-b4e4-14ea0e0cb79a.png)
![image](https://user-images.githubusercontent.com/7557597/216689810-a9a7a9f4-f9b7-42d4-a09a-11b0db15257e.png)


## ⚠️ To-do Before Merge



## 📝 Checklist
